pim-tool-back/src/main/java/pilotproject/Project_Management_ELCA/model/entity/Group.java

```diff
@@ -0,0 +1,46 @@
+package pilotproject.Project_Management_ELCA.model.entity;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.util.Objects;
+import java.util.Set;
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Table(name = "\"Group\"")
+public class Group {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false)
+    @Version
+    private int version;
+
+    @OneToOne(cascade = CascadeType.ALL)
+    @JoinColumn(name = "group_leader_id")
+    private Employee groupLeader;
+
+    @OneToMany(mappedBy = "group")
+    @JsonIgnore
+    private Set<Project> projectList;
```

Comment:
[S] entity `Project` does not override equality check, you need to override to ensure the correctness of Set.